docs/_config.yml

```diff
@@ -0,0 +1,9 @@
+title: Kadok
+remote_theme: pmarsceill/just-the-docs
+description: "Where's Kadoc? Is he well hidden?"
+footer_content: "Kadok is maintained by the team of \"Les Petits Pédestres\" under the <a href=\"https://github.com/Terag/kadok/blob/master/LICENSE\">GNU GENERAL PUBLIC LICENSE Version 3.</a>"
+color_scheme: nil
```

Comment:
why is there no `color_scheme` ? is it mandatory to specify it as `nil` ?

Reply:
From the theme [documentation](https://github.com/pmarsceill/just-the-docs/blob/master/_config.yml):
> Color scheme currently only supports "dark" or nil (default)

It is not very clear at first, a comment could be added for future modification.